content/foundations/responsive.mdx

```diff
@@ -41,23 +41,15 @@ Todo: Link to Figma Templates
 
 ### Pointing device
 
-Use the device characteristics through [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries) to adapt the experience accordingly for different device scenarios. The table below shows some possible device scenarios using `pointer` and `hover` media features:
+Consider multimodal input capabilities when designing interfaces. Many devices possess the ability to respond to cursor, touch, keyboard, voice, and other forms of input simultaneously. Additionally, many devices have the ability to have their input augmented in ways you may not initially expect. For example, many smartphones can support mouse and keyboard-based input, although their browsers may not be able to detect the presence of this augumentation.
 
-| &nbsp;             |                `pointer: coarse`                 |   `pointer: fine`    |
-| ------------------ | :----------------------------------------------: | :------------------: |
-| **`hover: none`**  |            Smartphones, touch screens            | Stylus-based screens |
-| **`hover: hover`** | Virtual reality headsets, video-game controllers |   Mouse, touch pad   |
+Because of this, pointing device media queries such as `coarse` or `fine` are unreliable. Avoid relying on this type of query to conditionally modify parts of an interface.
 
 #### Minimum target
 
-When creating interactive targets in your design, ensure that they are large enough to easily tap or click on. The AA accessibility standard GitHub strives for requires a minimum target size of `32px` for touch `pointer: coarse` and `24px` for mouse/stylus `pointer: fine`.
+When creating interactive targets in your design, ensure that they are large enough to easily tap or click on. The AA accessibility standard GitHub strives for requires a minimum target size of `24px`.
```

Comment:
This is great! So that means even our [smallest control sizing tokens](https://primer.style/primitives/storybook/?path=/story/size-functional-control--x-small) actually cover this without the additional media query I believe?

Reply:
I think so, yes!